types/diff/index.d.ts

```diff
@@ -142,6 +142,7 @@ export interface Hunk {
     newStart: number;
     newLines: number;
     lines: string[];
+    linedelimiters?: string[];
```

Comment:
This shouldn't be optional.

Reply:
Ok, will fix. I originally made it optional for backwards compatibility, but I just realized that the property in question was added three years ago anyway.

Reply:
Fixed with 3a4a4bb.